public/app/features/admin/LiveAdmin.tsx

```diff
@@ -42,11 +42,19 @@ export class LiveAdmin extends PureComponent<Props, State> {
       const msg = {
         line: text,
       };
+      const idx = channel.indexOf('/');
+      const plugin = channel.substring(0, idx);
+      const path = channel.substring(idx + 1);
```

Comment:
If `plugin` and `path` can't contain forward slashes (is this validated anywhere?), then this can be replaced with
```suggestion
      const [plugin, path] = channel.split('/');
```

Reply:
plugin can not contain slash, but path can and should be anything -- using a slash in the path should be normal.  For example in #27583 the channel should be `grafana/dashboard/${uid}`